mule-deployer/src/main/java/org/mule/tools/client/agent/AgentClient.java

```diff
@@ -57,4 +58,22 @@ public void undeployApplication(String appName) {
     }
   }
 
+  public String getMuleRuntimeVersion() {
```

Comment:
wouldn't it make more sense to just have a getAgentInfo and return the AgentInfo POJO? 

Currently we have no reason for this to be a problem, and granted we've done this in other clients BUT in this specific scenario it kind of sound better to just return AgentInfo and have the user to get the data from it wouldn't you say ?

Reply:
also if you agree please create getters and setters for the AgentInfo POJO (we'll do the same with the other ones in due time)

Reply:
Done!